app/templates/admin/courseManagement.html

```diff
@@ -0,0 +1,87 @@
+{% extends "base.html" %}
+{% block scripts %}
+  {{super()}}
+    <script src="/static/js/courseManagement.js"></script>
+{% endblock %}
+
+{% block app_content %}
+
+<div id="content">
+  <h1 class="text-center pt-1 pb-4">Service Learning Course Proposal Management</h1>
+  <div class="text-center">
+    <form id="termSelector" action="" method="post">
+      <select class="w-25" id="selectTerm" onchange="formSubmit(this.value)">
+          {% for term in terms %}
+            <option value="{{term.id}}" {{ "selected" if term.id == termId }}> {{ term.description }}</option>

```

Comment:
Does this actually work? Because `termId` is an object here, not a number. If it does work, you're just getting lucky that the object is smart enough to convert itself to a number for the operation (or rather, that it has implemented the equality operator with an integer by comparing to its ID)